server/models/Collective.js

```diff
@@ -553,8 +554,11 @@ const Collective = sequelize.define(
 
     tags: {
       type: DataTypes.ARRAY(DataTypes.STRING),
+      validate: {
+        validateTags,
+      },
```

Comment:
As I remember it, validators run on each model update - not just when the field changed. So as mentioned when we reviewed this together during the call, we need a strategy for existing collectives that don't pass this validation.

Reply:
My thinking was that when/if they try to update something on the Collective model, they would be prompted to fix it essentially. But I was then assuming the user would be trying to edit something on the Collective info settings page (and then could easily fix their tags)

But maybe the Collective model is updated as a side effect of doing other unrelated things, and having the tag validation error out then wouldn't be great.